src/makeCalculator.js

```diff
@@ -36,8 +36,43 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      return calculator.result + number;
+    },
+
+    subtract(number) {
+      return calculator.result - number;
+    },
+
+    divide(number) {
+      if (number !== 0) {
+        return calculator.result / number;
+      }
+    },
```

Comment:
Now when there is 0, nothing will happen, you need to show a message that you cannot divide by 0